src/App.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable max-len */
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [perPage, setPerPage] = useState<number>(5);
```

Comment:
```suggestion
  const [currentPage, setCurrentPage] = useState(1);
  const [perPage, setPerPage] = useState(5);
```

you don't need to add type explicitly here as TS is already aware about type by your initial state